client/flutter/lib/main.dart

```diff
@@ -143,4 +148,33 @@ class _MyAppState extends State<MyApp> {
       ),
     );
   }
+
+  @override
+  void didChangeAppLifecycleState(AppLifecycleState state) async {
+    switch (state) {
+      case AppLifecycleState.resumed:
+        if (await UserPreferences().getTermsOfServiceAccepted()) {
+          // ignore: unawaited_futures
+          ContentLoading().preCacheContent(getLocale());
+        }
+        break;
+      case AppLifecycleState.inactive:
+      case AppLifecycleState.paused:
+      case AppLifecycleState.detached:
+        break;
+    }
+  }
+
+  /// Construct the Locale from the Intl locale string.
+  /// This allows us to get the Locale outside of the main build context.
+  Locale getLocale() {
+    var parts = Intl.getCurrentLocale().split('_');
```

Comment:
What exactly is the difference between this and `Locale(Intl.getCurrentLocale())`?

Reply:
`Intl.getcurrentLocale()` returns the full locale string.  So the previous usage of `Locale(Intl.getCurrentLocale())` was incorrect as it was passing e.g. `en_US` as the language code and `null` as the country code.  This resulted in trying to load files e.g. `protect_yourself.en_US_null.yaml`.  I believe this fixes the problem and I've subbed it in for the previous usage.